dll/win32/shell32/res/rgs/controlpanel.rgs

```diff
@@ -73,3 +74,28 @@ HKCR
 		}
 	}
 }
+
+HKCR
+{
+	NoRemove CLSID
+	{
+		ForceRemove {0DF44EAA-FF21-4412-828E-260A8728E7F1} = s 'Taskbar and Start Menu'
+		{
+			val '{305CA226-D286-468e-B848-2B2E8E697B74} 2' = d '1'
+			val InfoTip = e '@%%SystemRoot%%\system32\SHELL32.dll,-30348'
+			val LocalizedString = e '@%%SystemRoot%%\system32\SHELL32.dll,-32517'
+			DefaultIcon = e '%%SystemRoot%%\system32\SHELL32.dll,-40'
+			Shell
+			{
+				Open
+				{
+					Command = s 'rundll32.exe shell32.dll,Options_RunDLL 1'
+				}
+			}
+			ShellFolder
+			{
+				val Attributes = d '0'
```

Comment:
Just to understand, here we don't want the same list of attributes as the `'Control Panel'` entry above?

Reply:
No, the Control Panel is a folder, this is a "fake file". Specifically, we don't want SFGAO_FOLDER to be set.